src/org/jitsi/impl/neomedia/transform/dtls/DtlsControlImpl.java

```diff
@@ -518,14 +643,14 @@ public DtlsControlImpl(boolean srtpDisabled)
 
         // The methods generateKeyPair(), generateX509Certificate(),
         // findHashFunction(), and/or computeFingerprint() may be too CPU
-        // intensive to invoke for each new DtlsControlImpl instance. That's why
-        // we've decided to reuse their return values within a certain time
+        // intensive to invoke for each new DtlsControlImpl instance. That's
+        // why we've decided to reuse their return values within a certain time
         // frame. Attempt to retrieve from the cache.
         synchronized (DtlsControlImpl.class)
         {
             certificateInfo = certificateInfoCache;
             if (certificateInfo == null
-                    || certificateInfo.timestamp + ONE_DAY
+                    || certificateInfo.timestamp + CERT_CACHE_EXPIRE_TIME
```

Comment:
What do you think of this: dropping CERT_VALIDITY_TIME and keeping CERT_CACHE_EXPIRE_TIME, and padding the validity by ONE_DAY to either side of the cache expiration?
Essentially the same idea, right? Except making clear to the admin that it is a cache that he/she is tuning. And then the software adjusts the certificate expirations to keep the admin out of trouble.